dev-tools/packaging/templates/common/README.md.tmpl

```diff
@@ -24,4 +24,4 @@ for the full {{.BeatName | title}} documentation.
 
 ## Release notes
 
-https://www.elastic.co/guide/en/beats/libbeat/{{ beat_doc_branch }}/release-notes-{{.Version}}.html
+https://www.elastic.co/guide/en/fleet/{{ beat_doc_branch }}/release-notes-{{.Version}}.html
```

Comment:
Actually we need to fix the references for `beat_doc_branch` above. The are getting set to `main` instead of `master` as well.

Example `https://www.elastic.co/guide/en/beats/{{.BeatName}}/{{ beat_doc_branch }}/index.html`

Reply:
We had the same problem in Beats https://github.com/elastic/beats/pull/34366

Reply:
We are planning to switch the docs over to main when we get all of the hard-coded master links fixed. But for now, maybe we can just set up redirects for any links that you include in this file. I would use links in files like this sparingly because they are not validated.

Reply:
Changing main > master won't fix the other problems with the readme, though. The links and commands are for Beats, not Elastic Agent.

Reply:
The links will work, but the content in that readme is not accurate. You don't run the setup command when using Elastic Agent, for example.

Reply:
That is entirely right, I needed to expand the entire file. This is just copied from Beats directly (this whole repository was surgically removed from Beats in 8.x).

What we actually want to do is fix this text but we aren't sure where it lives:

```
# Welcome to Elastic-Agent 8.6.0
Agent manages other beats based on configuration provided.

## Getting Started
To get started with Elastic-Agent, you need to set up Elasticsearch on
your localhost first. After that, start Elastic-Agent with:
    ./elastic-agent -c elastic-agent.yml -e
This will start Elastic-Agent and send the data to your Elasticsearch
instance. To load the dashboards for Elastic-Agent into Kibana, run:
   ./elastic-agent setup -e
For further steps visit the
[Quick start](https://www.elastic.co/guide/en/beats/elastic-agent/main/elastic-agent-installation-configuration.html) guide.

## Documentation
Visit [[Elastic.co](http://elastic.co/) Docs](https://www.elastic.co/guide/en/beats/elastic-agent/main/index.html)
for the full Elastic-Agent documentation.

## Release notes
https://www.elastic.co/guide/en/beats/libbeat/main/release-notes-8.6.0.html
```

Reply:
🤦 re-reading the request on Slack we *do* use this text verbatim, and the actual commands are wrong not just the links as DeDe pointed out.

Maybe this file should just be a link to the official agent documentation? It is extremely out of date, like I said this was brought in when this repository migrated out of Beats and appears to have never been updated.

Reply:
++ I agree with @cmacknz. This readme sends the user down the road of installing a standalone agent, which we consider to be an advanced use case. I would just send users https://www.elastic.co/guide/en/fleet/master/index.html (making sure the branch is set correctly to the version).

Reply:
Thanks.  I'll make the necessary changes to this PR.

Reply:
Done; please review when you get a chance, @cmacknz and @dedemorton.  Thanks!